test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java

```diff
@@ -441,11 +442,16 @@ public void TestQueryBatch(String inputFileName) throws SQLException, ClassNotFo
             expectedFile = new File(parallelQueryGeneratedFilesDirectoryPath + outputFileName + ".out");
             nonDefaultServerCollationExpectedFile = new File(parallelQueryGeneratedFilesDirectoryPath + "non_default_server_collation/" + serverCollationName + "/" + outputFileName + ".out");
         }
+        else if (isSingleDbMode && checkSingleDbModeExpected){
+            expectedFile = new File(generatedFilesDirectoryPath + "single_db/" + outputFileName + ".out");
+            nonDefaultServerCollationExpectedFile = new File(generatedFilesDirectoryPath + "single_db/" + "non_default_server_collation/" + serverCollationName + "/" + outputFileName + ".out");
+        }
```

Comment:
What happens when input file has both `-- parallel_query_expected` & `-- single_db_mode_expected`

Reply:
We will not run single-db mode in parallel query ?

Reply:
No, there is no plan to run single-db mode in parallel query.